.solhintignore

```diff
@@ -0,0 +1,19 @@
+# Dependency directories
```

Comment:
suggestion: remove `.solhintignore`

SolHint is not used, there' s no need for it's ignore file